independent-publisher-connectors/De Lijn/apiProperties.json

```diff
@@ -0,0 +1,24 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "constraints": {
+            "clearText": false,
+            "required": "true",
+            "tabIndex": 2
+          },
+          "description": "The api key from DeLijn",
+          "displayName": "Api key",
+          "tooltip": "Provide your KEY"
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFF",

```

Comment:
@Lschockaert please change this to "#da3b01". please follow this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json)

Reply:
I saw this in the pull request sentences but I guess I didn't fully understand the certified vs independent publisher thing.